docling_jobkit/ray_job/main.py

```diff
@@ -4,7 +4,7 @@
 from typing import Optional
 from urllib.parse import urlparse, urlunsplit
 
-import ray
+import ray  # type: ignore
```

Comment:
is this needed? are you sure you installed the local dev environment with `uv sync --all-extras`?